### DISCUSSION TITLE
bootleg to test

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#210] [Show and tell]
Might be more of them , randomly  tested a couple of them and worked

http://93.190.139.34:8278/streams/d/Hbofamily/playlist.m3u8
http://93.190.139.34:8278/streams/d/Hbohits/playlist.m3u8
http://93.190.139.34:8278/streams/d/Foxhd/playlist.m3u8
http://93.190.139.34:8278/streams/d/Foxmovie/playlist.m3u8
http://93.190.139.34:8278/streams/d/Foxfamily/playlist.m3u8
http://93.190.139.34:8278/streams/d/Axn/playlist.m3u8
http://93.190.139.34:8278/streams/d/Natgeopeople/playlist.m3u8
http://93.190.139.34:8278/streams/d/Cn/playlist.m3u8
http://93.190.139.34:8278/streams/d/Disneyxd/playlist.m3u8
http://93.190.139.34:8278/streams/d/Axn/playlist.m3u8
http://93.190.139.34:8278/streams/d/Tlc/playlist.m3u8
http://93.190.139.34:8278/streams/d/Afc/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhk1_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhkg_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhke_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nippontv_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/asahitv_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhk_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/jettv_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/tokyotv_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/fujitv_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/tokyomx_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhkeosaka_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhkbs1_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhkpremium_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/bs181_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/bsfuji_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/cna_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/bstbs_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/nhkosaka_japan/playlist.m3u8
http://93.190.139.34:8278/streams/d/cbs_korea/playlist.m3u8
http://93.190.139.34:8278/streams/d/ebs2_korea/playlist.m3u8
http://93.190.139.34:8278/streams/d/ebsplus2_korea/playlist.m3u8
http://93.190.139.34:8278/streams/d/sbsplus_korea/playlist.m3u8
http://93.190.139.34:8278/streams/d/TVRI/playlist.m3u8
http://93.190.139.34:8278/streams/d/Tvone/playlist.m3u8
http://93.190.139.34:8278/streams/d/Transtv/playlist.m3u8
http://93.190.139.34:8278/streams/d/Rcti/playlist.m3u8
http://93.190.139.34:8278/streams/d/Antv/playlist.m3u8
http://93.190.139.34:8278/streams/d/Kompass/playlist.m3u8
http://93.190.139.34:8278/streams/d/Trans7_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/metrotv_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/Sctv/playlist.m3u8
http://93.190.139.34:8278/streams/d/Indosiar/playlist.m3u8
http://93.190.139.34:8278/streams/d/Nettv/playlist.m3u8
http://93.190.139.34:8278/streams/d/Mnctv/playlist.m3u8
http://93.190.139.34:8278/streams/d/GlobalTV_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/BeritaSatu_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/TV9_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/CNNIndonesia_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/JTV_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/OChannel_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/Balitv/playlist.m3u8
http://93.190.139.34:8278/streams/d/BandungTV_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/TVMU_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/PijarTV_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/utv_eko/playlist.m3u8
http://93.190.139.34:8278/streams/d/MYTV_eko/playlist.m3u8

http://93.190.139.35:8278/streams/d/bbc1_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/bbc2_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/bbcworldnews_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/cbbc_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/itv1_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/itv2_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/Thrill/playlist.m3u8
http://93.190.139.35:8278/streams/d/One/playlist.m3u8
http://93.190.139.35:8278/streams/d/Weishimovie/playlist.m3u8
http://93.190.139.35:8278/streams/d/Weishilegend/playlist.m3u8
http://93.190.139.35:8278/streams/d/Animax/playlist.m3u8
http://93.190.139.35:8278/streams/d/cnn_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/Hits/playlist.m3u8
http://93.190.139.35:8278/streams/d/Hitsmovie/playlist.m3u8
http://93.190.139.35:8278/streams/d/Celestialindo/playlist.m3u8
http://93.190.139.35:8278/streams/d/Starplushd/playlist.m3u8
http://93.190.139.35:8278/streams/d/starmovies_raj/playlist.m3u8
http://93.190.139.35:8278/streams/d/Colorhd/playlist.m3u8
http://93.190.139.35:8278/streams/d/Zeeaction/playlist.m3u8
http://93.190.139.35:8278/streams/d/Zeecinemahd/playlist.m3u8
http://93.190.139.35:8278/streams/d/Zeetvhd/playlist.m3u8
http://93.190.139.35:8278/streams/d/starvijay/playlist.m3u8
http://93.190.139.35:8278/streams/d/Zeetamil/playlist.m3u8
http://93.190.139.35:8278/streams/d/Axn/playlist.m3u8
http://93.190.139.35:8278/streams/d/Suntv/playlist.m3u8
http://93.190.139.35:8278/streams/d/Ria_backup/playlist.m3u8
http://93.190.139.35:8278/streams/d/espn/playlist.m3u8
http://93.190.139.35:8278/streams/d/more4_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/freesports_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/skynews_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/picktv_antik/playlist.m3u8
http://93.190.139.35:8278/streams/d/euronews_antik/playlist.m3u8

http://93.190.139.36:8278/streams/d/hkcnews_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/hk03_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/jade_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/j2_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/j5_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/pearl_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/hkphoenix_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/phoenixtv_hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/phoenixinfo_hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/tvbcaijing_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/tvbxinghe_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/9ae6d37d9a/playlist.m3u8
http://93.190.139.36:8278/streams/d/viu_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/viu1_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/rhk31_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/rhk32_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/chcfamily/playlist.m3u8
http://93.190.139.36:8278/streams/d/chchdmovie/playlist.m3u8
http://93.190.139.36:8278/streams/d/chcaction/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv1/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv2/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv3/playlist.m3u8
http://93.190.139.36:8278/Cctv4/streams/d/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv4/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv5/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv5plus/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv6/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv7/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv8hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv9hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv10hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv11/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv12/playlist.m3u8
http://93.190.139.36:8278/cctv13/playlist.m3u8
http://93.190.139.36:8278/cctv14/playlist.m3u8
http://93.190.139.36:8278/cctv15/playlist.m3u8
http://93.190.139.36:8278/streams/d/dongfangweishi/playlist.m3u8
http://93.190.139.36:8278/streams/d/gdnanfang2/playlist.m3u8
http://93.190.139.36:8278/streams/d/Jiangsu/playlist.m3u8
http://93.190.139.36:8278/streams/d/jiangsu_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/Zhejiang/playlist.m3u8
http://93.190.139.36:8278/streams/d/zhejiang_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/shenzhen/playlist.m3u8
http://93.190.139.36:8278/streams/d/hubei/playlist.m3u8
http://93.190.139.36:8278/streams/d/Hunan/playlist.m3u8
http://93.190.139.36:8278/streams/d/hunan_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/fengyunzuqiu/playlist.m3u8
http://93.190.139.36:8278/streams/d/fujian/playlist.m3u8
http://93.190.139.36:8278/streams/d/xingkong/playlist.m3u8
http://93.190.139.36:8278/streams/d/beijing/playlist.m3u8
http://93.190.139.36:8278/streams/d/guangxi/playlist.m3u8
http://93.190.139.36:8278/streams/d/sichuan_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/zhujiang/playlist.m3u8
http://93.190.139.36:8278/streams/d/gdsports/playlist.m3u8
http://93.190.139.36:8278/streams/d/xinshijue/playlist.m3u8
http://93.190.139.36:8278/streams/d/disneych_glo/playlist.m3u8
http://93.190.139.36:8278/streams/d/disneyjunior_glo/playlist.m3u8
http://93.190.139.36:8278/streams/d/abcnews_glo/playlist.m3u8

**Comment by jkemp814** [comment]
None play in first group: http://93.190.139.34

These will not play in second group: http://93.190.139.35
http://93.190.139.35:8278/streams/d/Weishimovie/playlist.m3u8
http://93.190.139.35:8278/streams/d/Weishilegend/playlist.m3u8
http://93.190.139.35:8278/streams/d/Ria_backup/playlist.m3u8
http://93.190.139.35:8278/streams/d/espn/playlist.m3u8

Third group: http://93.190.139.36
These would not play:
http://93.190.139.36:8278/streams/d/hkcnews_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/hk03_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/jade_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/j2_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/j5_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/pearl_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/hkphoenix_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/phoenixtv_hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/phoenixinfo_hd/playlist.m3u8
http://93.190.139.36:8278/streams/d/tvbcaijing_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/tvbxinghe_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/9ae6d37d9a/playlist.m3u8
http://93.190.139.36:8278/streams/d/viu_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/viu1_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/rhk31_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/chchdmovie/playlist.m3u8
http://93.190.139.36:8278/streams/d/chcaction/playlist.m3u8
http://93.190.139.36:8278/Cctv4/streams/d/playlist.m3u8
http://93.190.139.36:8278/streams/d/cctv4/playlist.m3u8
http://93.190.139.36:8278/cctv13/playlist.m3u8
http://93.190.139.36:8278/cctv14/playlist.m3u8
http://93.190.139.36:8278/cctv15/playlist.m3u8
http://93.190.139.36:8278/streams/d/dongfangweishi/playlist.m3u8
http://93.190.139.36:8278/streams/d/gdnanfang2/playlist.m3u8
http://93.190.139.36:8278/streams/d/Jiangsu/playlist.m3u8
http://93.190.139.36:8278/streams/d/jiangsu_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/Zhejiang/playlist.m3u8
http://93.190.139.36:8278/streams/d/zhejiang_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/Hunan/playlist.m3u8
http://93.190.139.36:8278/streams/d/hunan_twn/playlist.m3u8
http://93.190.139.36:8278/streams/d/fengyunzuqiu/playlist.m3u8
http://93.190.139.36:8278/streams/d/xingkong/playlist.m3u8
http://93.190.139.36:8278/streams/d/xinshijue/playlist.m3u8
http://93.190.139.36:8278/streams/d/disneych_glo/playlist.m3u8
http://93.190.139.36:8278/streams/d/disneyjunior_glo/playlist.m3u8

**Comment by BellezaEmporium** [comment]
Z-DASH on port 8277 Auth Required (unknown)
Port 8283 will look like some monitoring.
![image](https://user-images.githubusercontent.com/30985701/136617032-bac407ea-27a2-4433-ba00-cb303dbc9ddf.png)


Pattern sounds the same than other channels we got (look at the Indonesian playlist, the Astro links)
